sensu/map.jinja

```diff
@@ -33,7 +33,8 @@
             'mailer_smtp_address': 'localhost',
             'mailer_smtp_port': '25',
             'mailer_smtp_domain': False,
-            'mailer_admin_gui': 'http://sensu.local'
+            'mailer_admin_gui': 'http://sensu.local',
+            'level-2-support_url': 'http://10.0.2.2:9393/notify'
```

Comment:
We should probably remove this as the default (as it won't work - it was only a development instance) and set it to `False` like we do for hipchat_* values.